packages/design-system/src/connect/components/document-toolbar/document-toolbar.tsx

```diff
@@ -15,6 +14,12 @@ import { twMerge } from "tailwind-merge";
 import { DocumentTimeline } from "../document-timeline/document-timeline.js";
 import { useDocumentUndoRedo } from "./utils/use-document-undo-redo.js";
 
+// TODO: Remove this when timeline analytics is available
+// import { useDocumentTimeline } from "@powerhousedao/reactor-browser/analytics";
```

Comment:
Importing `useDocumentTimeline` causes pglite to be added to the bundle. Commenting out for a later fix, as the feature itself is not functioning anyway